tools/wake-format/emitter.cpp

```diff
@@ -1263,6 +1250,12 @@ wcl::doc Emitter::walk_binary(ctx_t ctx, CSTElement node) {
     parts = collect_right_binary(op_token, node);
   }
 
+  if (ctx.explode_option == ExplodeOption::Prevent) {
+    MEMO_RET(select_best_choice({
```

Comment:
why use select_best_choice when there is only one option?

Reply:
because I didn't thing about it lol

Reply:
hey if you only get it one choice it makes the selection super easy